### PR TITLE
jenkins: parameterize container runtime when fetching logs and increase disk size

### PR DESCRIPTION
**Pull request by maelk**
Right now, fetching the logs assume docker, hence does not fetch local containers logs when they are run with podman. This fixes this issue.

This PR also increases the disk size of jenkins vm to 200GB, to allow the nodes to use more of their 50GB disks.

### CODE REVIEW

File: jenkins/scripts/files/run_fetch_logs.sh [hunk 1]
```diff
@@ -27,13 +34,13 @@ do
...
-for DOCKER_CONTAINER in $DOCKER_CONTAINERS
+mkdir -p "${LOGS_DIR}/${CONTAINER_RUNTIME}"
+LOCAL_CONTAINERS="$("${CONTAINER_RUNTIME}" ps --format "{{json .}}" | jq -r '.Names')"
+for LOCAL_CONTAINER in $LOCAL_CONTAINERS
 do
-  mkdir -p "${LOGS_DIR}/docker/${DOCKER_CONTAINER}"
-  docker logs "$DOCKER_CONTAINER" > "${LOGS_DIR}/docker/${DOCKER_CONTAINER}/stdout.log" \
-  2> "${LOGS_DIR}/docker/${DOCKER_CONTAINER}/stderr.log"
+  mkdir -p "${LOGS_DIR}/${CONTAINER_RUNTIME}/${LOCAL_CONTAINER}"
+  "${CONTAINER_RUNTIME}" logs "$LOCAL_CONTAINER" > "${LOGS_DIR}/${CONTAINER_RUNTIME}/${LOCAL_CONTAINER}/stdout.log" \
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_fetch_logs.sh:42]: For some reason there are no logs saved from podman

Comment (maelk) [reply jenkins/scripts/files/run_fetch_logs.sh:42]: that should be fixed now. Podman didn't like the template string

Comment (maelk) [reply jenkins/scripts/files/run_fetch_logs.sh]: and was missing sudo :/

**Pull request merged**